packages/ag-charts-enterprise/src/features/annotations/annotationOptionsToolbar.css

```diff
@@ -25,6 +25,19 @@
     background: var(--color);
 }
 
+.ag-charts-annotations__color-picker-button--multi-color::after {
+    background: linear-gradient(
```

Comment:
It might be a bit nicer to factor out the background so the logic of how the `::after` element works is a bit clearer

```
.ag-charts-annotations__color-picker-button {
  --emblem: var(--color);
}

.ag-charts-annotations__color-picker-button--multi-color {
  --emblem: linear-gradient(...)
}

.ag-charts-annotations__color-picker-button::after {
  ...
  background: var(--emblem);
}
```